config/_shared.json

```diff
@@ -180,7 +180,8 @@
 		"onboarding-with-email",
 		"setup-site",
 		"do-it-for-me",
-		"importer"
+		"importer",
+		"woocommerce-install"
```

Comment:
Good job finding this one! It was pretty non-obvious to me when I was first looking for it 😭